saleor/graphql/checkout/mutations.py

```diff
@@ -50,6 +49,8 @@
 from ..shipping.types import ShippingMethod
 from .types import Checkout, CheckoutLine
 
```

Comment:
Remove new line.

Reply:
This was not altered. Should the line be removed anyway?